### DISCUSSION TITLE
Question about test

### DISCUSSION THREAD

**Discussion by looksgoood** [oxigraph/oxigraph#422] [Q&A] [answered]
Hello.
I'm new on oxigraph and graph db.
I want to know the support ratio of oxigraph ragarding SPARQL 1.1 query standard.
I think in this repo already have to run testsuite for SPARQL 1.1 query test. (https://github.com/oxigraph/oxigraph/tree/main/testsuite)
But when I run cmd with ``` cargo test ``` it is not run properly.
Does anyone know how to run this test or is there any other way to check that?
Thanks.

**Comment by Tpt** [comment, marked as answer]
Hi @looksgoood!

There are two ways to run the SPARQL 1.1 query testsuite.

1. It is evaluated with rust tests as you found. Some tests that do not pass are ignored (see [the code](https://github.com/oxigraph/oxigraph/blob/main/testsuite/tests/sparql.rs#L75)). [Here is how the CI runs the tests](https://github.com/oxigraph/oxigraph/blob/d8fa540b97e12c20a2c011590d8e9caf214d8528/.github/workflows/tests.yml#L123) (don't forget to clone also git submodules).
2. Use the test runner that will output a report in Turtle. For the SPARQL 1.0 testsuite:
```bash
cd testsuite
cargo run -- https://w3c.github.io/rdf-tests/sparql/sparql10/manifest.ttl > sparql10-report.ttl
```
